### PR TITLE
Minimal setup for the new authentication which supports both basic an…

### PR DESCRIPTION
**Pull request by worukan**
…d token authentication. Changed auth endpoint to token as this name reflects its function better than the previous one.

### CODE REVIEW

File: dds_web/api/user.py [hunk 1]
```diff
@@ -5,44 +5,40 @@
 ###############################################################################
 
 # Standard library
-import datetime
+from datetime import datetime, timedelta
```
Comment (inaod568) [inline dds_web/api/user.py:8]: We (dev group) decided on only importing modules, not functions and classes, if not absolutely necessary

File: dds_web/api/user.py [hunk 2]
```diff
@@ -5,44 +5,40 @@
...
 
     try:
         token = jwt.encode(
             {
                 "user": username,
-                "project": {"id": project_id, "verified": project_access, "permission": permission},
-                "exp": datetime.datetime.utcnow() + datetime.timedelta(hours=48),
+                "exp": datetime.utcnow() + timedelta(hours=48),
             },
             app.config["SECRET_KEY"],
```
Comment (inaod568) [inline dds_web/api/user.py:40]: Can we change this to `app.config.get("SECRET_KEY")` instead? Been thinking about this for a while.

File: dds_web/api/user.py (hunk 1 above)
Comment (worukan) [reply dds_web/api/user.py]: Done!

File: dds_web/api/user.py (hunk 2 above)
Comment (worukan) [reply dds_web/api/user.py]: Done!

**Review by inaod568 — APPROVED** [review]

**Pull request merged**